satellite/satellitedb/usercredits.go

```diff
@@ -110,7 +130,7 @@ func (c *usercredits) Create(ctx context.Context, userCredit console.UserCredit)
 	}
 
 	if rows != 1 {
-		return rewards.MaxRedemptionErr.New("create credit failed")
+		return rewards.MaxRedemptionErr.New("rows not equal to 1")
```

Comment:
I think if I were a consumer of this method, I wouldn't understand what `rows not equal to 1` meant as an error.